libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -3411,7 +3415,7 @@ string YulUtilFunctions::storageSetToZeroFunction(Type const& _type)
 			)")
 			("functionName", functionName)
 			("clearStruct", clearStorageStructFunction(dynamic_cast<StructType const&>(_type)))
-			("panic", panicFunction())
+			("panic", panicFunction(PanicCode::Generic))
```

Comment:
When would this and the above happen?

Reply:
Only though a bug in the compiler. We hope that the check is optimized away.

Reply:
Ah ok